app/themes/dashboard/ui/index.html

```diff
@@ -19,8 +19,8 @@
     <!-- endbuild -->
 
   </head>
-  <body ng-app="app" class="skin-black">
-
+  <body ng-app="app" class="skin-black sidebar-mini" ng-controller="SidebarController" sidebar-directive="sidebarState">
+    <toast></toast>
```

Comment:
Kinda wondering if this needs to be in index.html. If it could be in root.html, then you could use root controller.

Alternatively, how about just sticking sidebarState in $rootScope. Having this ng-controller on body really doesn't look right. Sounds like opening a hole for potential side-effects..